Documentation/ApiOverview/Backend/AccessControl/PasswordReset/Index.rst

```diff
@@ -1,72 +1,97 @@
-The link is valid for 2 hours, and a token is added to the link.
-If the password was provided correctly, it is updated for the user, and they can log in.
+Once the user has entered their email address, an email is sent with a link that
+allows to set a new password, which must consist of at least eight characters.
+The link is valid for 2 hours and a token is added to the link. If the password
+is entered correctly, it will be updated for the user and they can log in.
 
 Notes on security
 =================
 
-*  When having multiple users with the same email address, no reset functionality is provided.
-*  No information disclosure is built-in, so if the email address is not in the system, it is not disclosed to the outside.
-*  Rate limiting is activated for allowing three emails to be sent within 30 minutes per email address.
-*  Tokens are stored in the database but hashed again, just like the password.
-*  When a user has logged in successfully (e.g., because they remembered the password), the token is removed from the database, effectively invalidating all existing email links.
+*   When having multiple users with the same email address, no reset
+    functionality is provided.
+*   No information disclosure is built-in, so if the email address is not in the
+    system, it is not disclosed to the outside.
+*   Rate limiting is enabled so that three emails can be sent per email address
+    within 30 minutes.
+*   Tokens are stored in the database but hashed again just like the password.
+*   When a user has logged in successfully (for example, because they remembered
+    the password), the token is removed from the database, effectively
+    invalidating all existing email links.
 
 .. index:: Backend; passwordReset
 .. index:: Backend; passwordResetForAdmins
 
 Global configuration
 ====================
 
-The feature is active by default and can be deactivated entirely via the
+The feature is enabled by default and can be deactivated entirely via the
 system-wide configuration option:
 
-:php:`$GLOBALS['TYPO3_CONF_VARS']['BE']['passwordReset']`
+..  code-block:: php
 
-Optionally it is possible to restrict this feature to non-admins only by setting the following system-wide option to "false".
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['passwordReset'] = false;
 
-:php:`$GLOBALS['TYPO3_CONF_VARS']['BE']['passwordResetForAdmins']`
+Optionally, it is possible to restrict this feature to non-admins only by setting
+the following system-wide option to :php:`false`.
 
-Both options are available to be configured within the :guilabel:`Maintenance Area > Settings`
-module or in the :guilabel:`Install Tool` but can be set manually via :file:`config/system/settings.php` or :file:`config/system/additional.php`.
+..  code-block:: php
+
+    $GLOBALS['TYPO3_CONF_VARS']['BE']['passwordResetForAdmins'] = false;
```

Comment:
```suggestion
..   code-block:: php
    :caption: config/system/additional.php | typo3conf/system/additional.php

    $GLOBALS['TYPO3_CONF_VARS']['BE']['passwordResetForAdmins'] = false;
```

Reply:
Nope, that was by intend. It can be set in settings.php and additional.php. And via the GUI.

Reply:
But the code as displayed here would only work in the `additional.php`, also the settings.php should not be edited manually. Therefore I think mentioning the `additional.php` here makes sense.